### ISSUE TITLE
[ACCESSIBILITY] Filter buttons are missing text

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#540]
On the explorer, the buttons to open the filter panel and close the filter panel are empty or missing value text. 

- What It Means: A button is empty or has no value text.
- Why It Matters: When navigating to a button, descriptive text must be presented to screen reader users to indicate the function of the button.
- What To Do: Place text content within the <button> element or give the <input> element a value attribute.
![image](https://user-images.githubusercontent.com/76966798/106818320-f9c58e00-662c-11eb-87f3-74335726d3b1.png)
